test/support/utils_sycl.h

```diff
@@ -367,5 +353,35 @@ namespace TestUtils
         return data;
     }
 
+#if __cplusplus >= 201703L
```

Comment:
opps.. Sorry, U mised that _ONEDPL_SYCL_NOINIT just for the tests..
In that case lets keep that macro, as Denis done.